-1 
Tested at: 321fa34faa9499bf2d6e3f8a3799003d3a92cfd9
 I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TCSCTPG.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TCompare.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TDEMON.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TDTTF.cc 
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TCSCTF.cc: In member function 'virtual void L1TCSCTF::analyze(const edm::Event&, const edm::EventSetup&)':
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TCSCTF.cc:853:49: error: expected '(' before 'csctflcts'
                         else if                 csctflcts -> Fill(bx, 2.5);
                                                 ^
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-0200/src/DQM/L1TMonitor/src/L1TCSCTF.cc:869:49: error: expected '(' before 'csctflcts'
                         else if                 csctflcts -> Fill(bx, 15.5);
                                                 ^
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8125/3574/summary.html

--- a/DQM/L1TMonitor/src/L1TCSCTF.cc
+++ b/DQM/L1TMonitor/src/L1TCSCTF.cc
@@ -850,7 +850,7 @@ void L1TCSCTF::analyze(const Event& e, const EventSetup& c)
                         else                    csctflcts -> Fill(bx, 4.5);
                     } else if(station==2) {
                         if(ring==1)             csctflcts -> Fill(bx, 3.5);
-                        else if                 csctflcts -> Fill(bx, 2.5);
+                        else                    csctflcts -> Fill(bx, 2.5);
                     } else if(station==3) {
                         if(ring==1)             csctflcts -> Fill(bx, 1.5);
                         else                    csctflcts -> Fill(bx, 0.5);
@@ -866,7 +866,7 @@ void L1TCSCTF::analyze(const Event& e, const EventSetup& c)
                         else                    csctflcts -> Fill(bx, 13.5);
                     } else if(station==2) {
                         if(ring==1)             csctflcts -> Fill(bx, 14.5);
-                        else if                 csctflcts -> Fill(bx, 15.5);
+                        else                    csctflcts -> Fill(bx, 15.5);
                     } else if(station==3) {
                         if(ring==1)             csctflcts -> Fill(bx, 16.5);
                         else                    csctflcts -> Fill(bx, 17.5);

